### PR TITLE
[ci]: Support azp for cisco 8000

### PR DESCRIPTION
**Pull request by xumia**
#### Why I did it
Setup Azure pipeline for cisco 8000.

#### How to verify it
Merge the code to the following repo, and setup azp for the repo.
https://github.com/Cisco-8000-sonic/sonic-buildimage

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [x] 202012
- [x] 202106

### CODE REVIEW

**Review by liushilongbuaa — APPROVED** [review]

**Comment by tahmed-dev** [comment]
@xumia when do you plan on completing this PR?

File: .azure-pipelines/official-build-cisco-8000.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        submodules: recursive
+        path: s/platform/cisco-8000
+        displayName: 'Checkout cisco-8000'
+      - task: DownloadGitHubRelease@0
+        inputs:
+          connection: cisco-connection
+          userRepository: Cisco-8000-sonic/platform-cisco-8000
+          defaultVersionType: specificTag
+          version: 202012-v0.2
+          itemPattern: 'artifactory-*.tar.gz'
```
Comment (judyjoseph) [inline .azure-pipelines/official-build-cisco-8000.yml:52]: Can we get the artifactory-*.tar.gz here based on the branch the az-pipeline is being run for ? 
If do we can use the same pipeline.yml file for getting cisco-build done for 202106 branch as well. The artifactiory we need in 202106 would be different.

Comment (xumia) [reply .azure-pipelines/official-build-cisco-8000.yml:57]: @judyjoseph , I have changed to use the value specified in cisco-8000.ini, do you think it is good now?

**Review by liushilongbuaa — APPROVED** [review]

**Pull request merged**